assets/css/stripeStyle.css

```diff
@@ -0,0 +1,110 @@
+* {
+    box-sizing: border-box;
+}
+  
+body,
+html {
+    background-color: #f6f9fc;
+    font-size: 13px;
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    margin: 0;
+}
+
+label {
+    color: #6b7c93;
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5864152)